examples/uptime_sla/test/UptimeSLA_test.js

```diff
@@ -7,33 +7,34 @@ import {
 } from './support/helpers'
 import {
   assertActionThrows,
+  bigNum,
   eth,
+  deploy,
   functionSelector,
-  getEvents,
   getLatestEvent,
   oracleNode,
   requestDataBytes,
-  requestDataFrom
+  requestDataFrom,
+  toWei,
 } from '../../../solidity/test/support/helpers'
-import { assertBigNum } from '../../../solidity/test/support/matchers'
 
 contract('UptimeSLA', () => {
   let Link = artifacts.require('LinkToken')
   let Oracle = artifacts.require('Oracle')
   let SLA = artifacts.require('UptimeSLA')
-  let specId = '4c7b7ffb66b344fbaa64995af81e355a'
+  let specId = '0x4c7b7ffb66b344fbaa64995af81e355a'
   let deposit = 1000000000
   let link, oc, sla, client, serviceProvider, startAt
 
   beforeEach(async () => {
     client = "0xf000000000000000000000000000000000000001"
     serviceProvider = "0xf000000000000000000000000000000000000002"
-    link = await Link.new()
+    link = await deploy('LinkToken.sol')
     oc = await Oracle.new(link.address, {from: oracleNode})
     sla = await SLA.new(client, serviceProvider, link.address, oc.address, specId, {
       value: deposit
     })
-    link.transfer(sla.address, web3.toWei(1, 'ether'))
+    link.transfer(sla.address, toWei('1', 'ether').toString())
```

Comment:
Do we need `.toString()` everywhere if it returns a string?

https://github.com/smartcontractkit/chainlink/blob/features/boot-wo-eth-connection/solidity/app/utils.js#L16-L18

```
    toWei: (eth) => {
      return (parseInt(eth.toString(), 10) * 10 ** 18).toString()
    },
```

Reply:
`toWei` from `helpers.js` returns a `BN`. My own preference, going forward, is to have a consistent return type for this, rather than respond with the type of the argument. I'd be OK with that being string, though.